lib/network_ssl/network_ssl.c

```diff
@@ -1,3 +1,11 @@
+#if defined(BSD)
```

Comment:
I'd prefer to omit this and say in BUILDING that adding -D__BSD_VISIBLE improves performance on BSD systems.  I did that with spiped back in 2013 when FreeBSD hadn't exposed MSG_NOSIGNAL.